### DISCUSSION TITLE
Profile Links

### DISCUSSION THREAD

**Discussion by kyoyacchi** [Docile-Alligator/Infinity-For-Reddit#1257] [Ideas]
Hi. Is it possible to show user's added connections on their profile?
Like in official reddit app.
![194377956-f311a753-3b69-427a-8198-82d723f732c6](https://user-images.githubusercontent.com/63583961/204515113-cc6c77c6-46f2-45c0-aa59-1d3ca2e5a0c7.jpg) (I don't use it, just to show links.)

**Comment by Wladefant** [comment]
Unfortunately that is not possible, because reddit does not provide the api for that function: https://www.reddit.com/r/redditdev/comments/ticwjl/is_it_possible_to_fetch_profile_links_with_praw/

**Reply by us3rname101** [reply]
Depends how you look at it. The api docs for the graphql is not public but with some chrome devtools you can see how it works. It is not advised to use it according to the [admins](https://www.reddit.com/r/redditdev/comments/fdh19d/comment/fjiic0c/) though.
After some testing I found it is done by sending a post request to https://gql.reddit.com/ with 
`{
  "id": "11a239b07f86",
  "variables": {
    "username": "username"
  }
}` body. the "11a239b07f86" seems to be hardcoded but might change overtime. Authorization is required so this wouldn't work in anonymous mode. But it could be done.
